doc/source/user_guide/hpc_slurm.rst

```diff
@@ -0,0 +1,609 @@
+cores, memory, GPUs) to jobs submitted by users.
+
+For more information regarding SLURM, visit `SLURM Documentation <slurm_docs_>`_.
+
+Basic concepts
+==============
+
+- **Nodes**: Individual computing servers within the cluster.
+- **Compute node**: A type of node used only for running processes.
+  It is not accessible from outside the cluster.
+- **Login nodes**: A type of node which is used only for login and job submission.
+  No computation should be performed on it. It is sometimes referred to as
+  'virtual desktop infrastructure' (VDI).
+- **Partition**: A logical grouping of nodes with similar characteristics
+  (for example CPU architecture, memory size). 
+- **Job**: A task submitted to SLURM for execution. 
+- **Queue**: A waiting area where jobs are held until resources become available. 
+- **Scheduler**: The component responsible for deciding which job gets executed
+  when and where.
+
+
+Regular job submission workflow
+===============================
+
+Login into the cluster
+----------------------
+
+You need access credentials and permissions to log in and submit jobs on the HPC cluster.
+Depending on the login node configuration, you can login using GUI based tools like VNC or just a terminal.
+
+For example, you can log in into a login node using the terminal:
+
+.. code-block:: console
+
+    user@machine:~$ ssh username@login-node-hostname
+
+
+Writing a SLURM Batch Script
+----------------------------
+
+A SLURM batch script is a shell script that specifies
+job parameters and commands to execute. Here's a basic example:
+
+**my_script.sh**
+
+.. code-block:: bash
+    
+    #!/bin/bash
+    #SBATCH --job-name=myjob
+    #SBATCH --nodes=1
+    #SBATCH --ntasks-per-node=4
+    #SBATCH --time=01:00:00
+
+    # Commands to run
+    echo "Hello, SLURM!"
+    srun my_executable
+
+
+This script is run using ``srun`` and ``sbatch``.
+Notice how the job configuration is detailed through comments in the
+file prefixed with ``#SBATCH``.
+For more information regarding the possible ``srun`` and ``sbatch``
+arguments visit `Slurm Workload Manager - srun <slurm_srun_>`_ and
+`Slurm Workload Manager - sbatch <slurm_sbatch_>`_.
+
+Submitting a Job
+----------------
+To submit a job, use the ``srun`` command followed by the name of
+the batch script:
+
+.. code-block:: console
+    
+    user@machine:~$ srun my_script.sh
+
+If you prefer to submit a batch job, you can use the ``sbatch`` command:
+
+.. code-block:: console
+    
+    user@machine:~$ sbatch my_script.sh
+
+You can specify each job setting using the command line. For example:
+
+.. code-block:: console
+
+    user@machine:~$ srun --nodes=2 my_script.sh
+
+.. warning:: **Command line arguments vs In-file arguments**:
+    The command line arguments do NOT overwrite the equivalent arguments written
+    in the bash file.
+    Hence, make sure that the argument you want to pass using the command line is
+    not present already in the bash file.
```

Comment:
@mcMunich @pmaroneh @mikerife  Can any of you confirm this statement? I couldn't find anything on the internet.